patches.js

```diff
@@ -30,13 +33,71 @@ const patchConfigManagerLoader =  () => {
       config.region = AWS_DEFAULT_REGION
       return config
     }
-    
-    console.log("Configuration Manager Patched")
+
+  } catch (error) {
+    console.error("Error:\t\tLocalstack Plugin unable to patch Configuration Manager", error)
+  }
+}
+
+// Patchs the utility that copy files from .ejs to replace the hardcoded AWS domains with LocalStack domains
+const patchCopyBatch = () => {
+  const newDomain = getLocalEndpoint().replace("https://", "").replace("http://", "")
+  const port = newDomain.split(":").pop()
+  const copyBatchPath = `${snapshot_path}@aws-amplify/cli-internal/lib/extensions/amplify-helpers/copy-batch`
+
+  try {
+
+    const copyBatchLib = require(copyBatchPath);
+    oldMethod = copyBatchLib.copyBatch
```

Comment:
```suggestion
    const oldMethod = copyBatchLib.copyBatch
```

On a related note - maybe we could configure an eslint code linter for the project? (in a follow-up PR..)